### DISCUSSION TITLE
Tagging offhour on RDS - RDS doesn't support , () ; characters

### DISCUSSION THREAD

**Discussion by stevelaver** [cloud-custodian/cloud-custodian#7318] [General]
I can't tag an Aurora instance with the required tag syntax. Error when trying to tag off(M-F,18)

Tag values may only contain unicode letters, digits, whitespace, or one of these symbols: _ . : / = + \ - @ (Service: AmazonRDS; Status Code: 400; Error Code: InvalidParameterValue; 

Is their an alternative syntax?

**Comment by kapilt** [comment]
you can't do offhours an aurora instance, just single zone rds instances, aka whats supported by rds stop/start api. that said it sounds like there might be additional tag restrictions on rds.

https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_StopInstance.html

**Comment by stevelaver** [comment]
Thanks, makes sense

**Comment by marpada** [comment]
Can we please reopen this issue?. The question is still valid on other non-Aurora RDS types, as it is a limitation of RDS tag values.

**Comment by jtroberts83** [comment]
@kapilt I continue to get this request from my customers to have RDS offhours tagging.  While I know it is available via a "workaround", getting the custodian code to do a replace on certain tag characters so that they match what c7n is looking for that seems like a somewhat easy win.

**Comment by pricy** [comment]
What is the workaround?
